demo-app/src/stories/SortControl.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import SortControl from '../components/sort-control/SortControl';
+
+export default {
+    title: 'SortControl',
+    component: SortControl
+}
+
+export const Basic = (args) => <SortControl {...args} />;
+Basic.args = {
+    currentSelection: "title",
+    onSortChange: "releaseDate"
```

Comment:
Here must be function. You see mistake because try to add string instead function.
I sent you some examples above. Let's try to fix this story yourself.